test/unit/cog_api/fake/users_test.exs

```diff
@@ -27,6 +27,26 @@ defmodule CogApi.Fake.UsersTest do
     end
   end
 
+  describe "show" do
+    it "returns the user" do
+      params = %{
+        first_name: "Charlie",
+        last_name: "Young",
+        email_address: "charlie@example.com",
+        username: "aide_to_potus",
+        password: "thesecretest",
+      }
+      {:ok, created_user} = Client.user_create(fake_endpoint, params)
+
+      {:ok, found_user} = Client.user_show(fake_endpoint, created_user.id)
+
+      assert found_user.first_name == params.first_name
```

Comment:
Would it make sense to only verify the `id` matches here? The other stuff should follow.